deps/build.jl

```diff
@@ -76,7 +76,13 @@ try # make sure deps.jl file is removed on error
 
     use_conda = dirname(python) == abspath(Conda.PYTHONDIR)
     if use_conda
-        Conda.add("numpy"; satisfied_skip_solve=true)
+        if Sys.iswindows() && Sys.WORD_SIZE == 32
+            # Conda is no longer distributed for 32-bit Windows
+            # and does not have satisified_skip_solve
+            Conda.add("numpy")
+        else
+            Conda.add("numpy"; satisfied_skip_solve=true)
```

Comment:
Shouldn't Conda.jl just ignore this flag in that case?

Reply:
In most cases, I would want Conda.jl to error rather than silenting ignoring the keyword. The caller should either anticipate the error or handle the error.

Here, I'm just trying to maintain backwards compatability for a very specific situation, 32-bit Windows, which upstream conda no longer supports.

Reply:
I'm not 100% convinced that an error is the behavior `Conda.add` users would want.

The `satisfied_skip_solve` flag in practice mostly seems like an optional hint to speed things up, as it is used here, and it's a bit ugly that callers of Conda.jl will have to know about and implement the 32-bit windows exception themselves.

Reply:
> Honestly, my preferred solution is to stop supporting conda on 32-bit Windows and Python 2.7 entirely due to lack of upstream support

I also see this as the best solution.